user-access-internal-model/src/test/java/no/unit/nva/useraccessservice/dao/UserDaoTest.java

```diff
@@ -174,42 +175,42 @@ void shouldConvertToDtoAndBackWithoutInformationLoss() {
-        List<RoleDb> invalidRoles = Collections.singletonList(invalidRole);
-        UserDao userDaoWithInvalidRole = UserDao.newBuilder()
-            .withUsername(SOME_USERNAME)
-            .withRoles(invalidRoles)
-            .build();
-
-        Executable action = userDaoWithInvalidRole::toUserDto;
-        RuntimeException exception = assertThrows(RuntimeException.class, action);
-        assertThat(exception.getCause(), is(instanceOf(InvalidEntryInternalException.class)));
-    }
-
-    @ParameterizedTest
-    @NullAndEmptySource
-    void toUserDbThrowsExceptionWhenUserDbContainsInvalidRole(String invalidRoleName)
-        throws InvalidEntryInternalException {
-        RoleDto invalidRole = RoleDto.newBuilder().withRoleName(SOME_ROLENAME).build();
-        invalidRole.setRoleName(invalidRoleName);
-        List<RoleDto> invalidRoles = Collections.singletonList(invalidRole);
-        UserDto userWithInvalidRole = UserDto.newBuilder().withUsername(SOME_USERNAME).withRoles(invalidRoles).build();
-
-        Executable action = () -> UserDao.fromUserDto(userWithInvalidRole);
-        RuntimeException exception = assertThrows(RuntimeException.class, action);
-        assertThat(exception.getCause(), is(instanceOf(InvalidEntryInternalException.class)));
-    }
+//    @ParameterizedTest(name = "fromUserDb throws Exception user contains invalidRole. Rolename:\"{0}\"")
+//    @NullAndEmptySource
+//    void fromUserDbThrowsExceptionWhenUserDbContainsInvalidRole(String invalidRoleName)
+//        throws InvalidEntryInternalException {
+//        RoleDb invalidRole = new RoleDb();
+//        invalidRole.setName(invalidRoleName);
+//        List<RoleDb> invalidRoles = Collections.singletonList(invalidRole);
+//        UserDao userDaoWithInvalidRole = UserDao.newBuilder()
+//            .withUsername(SOME_USERNAME)
+//            .withRoles(invalidRoles)
+//            .build();
+//
+//        Executable action = userDaoWithInvalidRole::toUserDto;
+//        RuntimeException exception = assertThrows(RuntimeException.class, action);
+//        assertThat(exception.getCause(), is(instanceOf(InvalidEntryInternalException.class)));
+//    }
+
+//    @ParameterizedTest
+//    @NullAndEmptySource
+//    void toUserDbThrowsExceptionWhenUserDbContainsInvalidRole(String invalidRoleName)
+//        throws InvalidEntryInternalException {
+//        RoleDto invalidRole = RoleDto.newBuilder().withRoleName(SOME_ROLENAME).build();
+//        invalidRole.setRoleName(invalidRoleName);
+//        List<RoleDto> invalidRoles = Collections.singletonList(invalidRole);
+//        UserDto userWithInvalidRole = UserDto.newBuilder().withUsername(SOME_USERNAME).withRoles(invalidRoles).build();
+//
+//        Executable action = () -> UserDao.fromUserDto(userWithInvalidRole);
+//        RuntimeException exception = assertThrows(RuntimeException.class, action);
+//        assertThat(exception.getCause(), is(instanceOf(InvalidEntryInternalException.class)));
+//    }
```

Comment:
Unsure if we still need these tests, all the unknown enum values will from now on be illegal.